sdk_extensions/zpages/src/main/java/io/opentelemetry/sdk/extensions/zpages/TracezZPageHandler.java

```diff
@@ -186,7 +187,10 @@ private static void emitSummaryTableCell(
     // If numOfSamples is smaller than 0, print the text "N/A", otherwise print the text "0"
     if (numOfSamples > 0) {
       out.print("<td class=\"align-center border-left-dark\"><a href=\"?");
-      out.print(PARAM_SPAN_NAME + "=" + URLEncoder.encode(spanName, "UTF-8"));
+      out.print(
+          PARAM_SPAN_NAME
+              + "="
+              + URLEncoder.encode(spanName.replace("+", /* Unicode for + */ "%2B"), "UTF-8"));
```

Comment:
I think you can use StandardCharsets.UTF-8 here, and avoid the exception that could result.

Reply:
Using `StandardCharsets.UTF_8.toString()` seems to still require the exception.

Reply:
oh shoot. the API that takes a CharSet is too new to use here.  never mind!

Reply:
Ah didn't know this behavior of urlencoder. In that case may want to use guava which I don't think has this problem

https://guava.dev/releases/19.0/api/docs/com/google/common/net/UrlEscapers.html#urlFormParameterEscaper()

Reply:
Ah sorry by "this behavior" I meant the inability to round trip space and plus reliably.